01-node-tutorial/answers/modules.js

```diff
@@ -10,17 +10,25 @@
 
 // NOTES
 
+// BUILT-IN MODULES
+// --OS
+// --PATH
+// --FS
+// --HTTP
+
 // CommonJS, every file is module (by default)
 // Modules - Encapsulated Code (only share minimum)
 
 // Tutorial code
 
-const names = require('./04-names');
-const sayHi = require('./05-utils');
-const data = require('./06-alt-flavor');
-// require('./07-mind-grenade');
+const names = require( './names' );
+const sayHi = require( './utils' );
+const data = require( './alternative-flavor' );
+// require( './mind-grenade' );
```

Comment:
I believe this line is required for the assignment

Reply:
Fixed issue